public/assets/i18n/en.json

```diff
@@ -140,6 +140,8 @@
   "sitnet_write_message": "Write message",
   "sitnet_message": "Message",
   "sitnet_minutes": "minutes",
+  "sitnet_hours": "hours",
```

Comment:
Näihin kantsii laittaa perään "EN" tai "SV" ihan vaan, että käännöstyöryhmä osaa sit aikanaan laittaa haluamansa käännökset tälle paikalle :)

Reply:
"sitnet_hours": "hours EN",
Niiku näin?